src/projects/lja/lja.cpp

```diff
@@ -75,6 +75,8 @@ int main(int argc, char **argv) {
     logger.info() << "Hello! You are running La Jolla Assembler (LJA), a tool for genome assembly from PacBio HiFi reads\n";
     logging::logGit(logger, dir / "version.txt");
     bool diploid = parser.getCheck("diploid");
+    bool trio = (parser.getListValue("maternal").size() != 0 && parser.getListValue("paternal").size() != 0);
```

Comment:
Add verify here with a message in case there are only maternal or only paternal reads

Reply:
+